pandas/tests/extension/json/test_json.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 
+from pandas.compat import PY36
```

Comment:
I think you can also import `PY2` here and replace `sys.version_info[0] == 2` on line 12 with it, as that is the exact definition of `PY2`.  That's the only remaining reference to `sys`, so you could then remove the `import sys` too.